Resources/public/js/views/ez-universaldiscoveryview.js

```diff
@@ -429,6 +431,7 @@ YUI.add('ez-universaldiscoveryview', function (Y) {
             container.setHTML(this.template({
                 title: this.get('title'),
                 multiple: this.get('multiple'),
+                startingLocationId: this.get('startingLocationId'),
```

Comment:
since you did not change anything in the corresponding, that does not seem to be needed